src/api/api.c

```diff
@@ -44,6 +44,7 @@ static struct {
 	{ "/api/groups",                            "/{name}",                    api_list,                              { false, true,  0               }, true,  HTTP_GET | HTTP_POST | HTTP_PUT | HTTP_DELETE },
 	{ "/api/lists",                             "/{list}",                    api_list,                              { false, true,  0               }, true,  HTTP_GET | HTTP_POST | HTTP_PUT | HTTP_DELETE },
 	{ "/api/info/client",                       "",                           api_info_client,                       { false, true,  0               }, false, HTTP_GET },
+	{ "/api/info/login",                        "",                           api_info_login,                        { false, true,  0               }, false, HTTP_GET },
```

Comment:
`Auth` is `false` here, but on the docs it says otherwise
![Screenshot at 2023-10-09 21-06-43](https://github.com/pi-hole/FTL/assets/26622301/bae793ed-9c60-4800-877b-dbc3a260e289)

(It's the same for `/info/client`and `/auth/totp`)